makefiles/pseudomodules.inc.mk

```diff
@@ -121,6 +121,10 @@ PSEUDOMODULES += log
 PSEUDOMODULES += log_printfnoformat
 PSEUDOMODULES += log_color
 PSEUDOMODULES += lora
+## @defgroup pseudomodule_libc_syscalls_gettimeofday libc_syscalls_gettimeofday
+## @brief Includes implementation of gettimeofday()
+##
+PSEUDOMODULES += libc_syscalls_gettimeofday
```

Comment:
would you shorten this to `libc_gettimeofday`? - (I think it would be better if it was shorter)

Reply:
Done